src/LaravelDebugbar.php

```diff
@@ -591,6 +591,12 @@ public function modifyResponse(Request $request, Response $response)
             } catch (\Exception $e) {
                 $app['log']->error('Debugbar exception: ' . $e->getMessage());
             }
+        } elseif ($app['config']->get('debugbar.inject', true) && $app['config']->get('debugbar.csp-compatible', true)) {
```

Comment:
You know you can just make a new commit to your branch and then it will change this pull request, right?